src/MsgPack/Serialization/SerializationContext.cs

```diff
@@ -117,6 +117,22 @@ public static SerializationContext Default
 
 		private readonly object _generationLock;
 
+		/// <summary>
+		/// The type member ignore list
+		/// </summary>
+		private readonly IDictionary<Type, IEnumerable<string>> _typesMemberIgnoreList = new Dictionary<Type, IEnumerable<string>>();
```

Comment:
@yfakariya , I didn't realize this. I will fix in next PR. Also i am kind of juggling where to put these since i have seen that you are divergining to XxxOptions, but couldn't think of better name.